### PR TITLE
other/tor.md: add a request that peers use LongLivedPorts

### LINKED ISSUE #73: Tor: note on long lived ports & socks authentication?
I have two suggestions and I am opening this issue so that I remember to create a PR at a better time:

* [ ] Add a note that Tor peers should be using long-lived ports to reduce chance of nodes going down.

> LongLivedPorts PORTS
> A list of ports for services that tend to have long-running connections (e.g. chat and interactive shells). Circuits for streams that use these
> ports will contain only high-uptime nodes, to reduce the chance that a node will go down before the stream is finished. Note that the list is
> also honored for circuits (both client and service side) involving hidden services whose virtual port is in this list. (Default: 21, 22, 706,
> 1863, 5050, 5190, 5222, 5223, 6523, 6667, 6697, 8300)
> ~~ `man torrc`

* ~~Use SOCKS authentication to cause Tor to isolate Yggdrasil traffic from other traffic.~~
    * ~~I guess this could also be used for adding a comment on who is the admin which sanitizing config file wouldn't remove?~~

> IsolateSOCKSAuth
> Don’t share circuits with streams for which different SOCKS authentication was provided. (For HTTPTunnelPort connections, this option looks
> at the Proxy-Authorization and X-Tor-Stream-Isolation headers. On by default; you can disable it with NoIsolateSOCKSAuth.)
> ~~ `man torrc`

~~Yggdrasil may support this by default as so does dnscrypt-proxy?~~

> `socks5://user:password@127.0.0.1:9050` should work. TBH, I haven't tried but the Go SOCK5 client code I use appears to support authentication out of the box.
> https://github.com/jedisct1/dnscrypt-proxy/issues/560#issuecomment-410690494

### PR DESCRIPTION
**Pull request by Mikaela**
Closes #73

### CODE REVIEW

**Comment by akklesed** [comment]
>to reduce the chance that a node will go down

Probably better wording would be "to increase node's availability" or "stability".
While it's true that services go down if circuit is broken, they come up back relatively quickly, and current wording kinda makes it sound like they don't.

**Comment by Mikaela** [comment]
Fixed, thanks :) 

I felt that line was a bit awkward, even if I was taking it from torrc, but I couldn't think of anything better.

File: other/tor.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Tor hidden service peers
 
+*It's recommended that Tor peers use Tor's [LongLivedPorts] (21, 22, 706,
```
Comment (Mikaela) [inline other/tor.md]: Another thing I am not entirely happy with is how I repat "Tor" twice here, does anyone have better wording suggestions?

Comment (cathugger) [reply other/tor.md]: >Tor peers

this is already page for Tor peers so you can skip that and just have "peers", I don't think that's going to be confusing.

Comment (Mikaela) [reply other/tor.md]: Fixed, thanks :)

**Comment by cathugger** [comment]
@neilalexander @Arceliar poke I guess I should wait for approval from you guys before merging, though this looks good to me as I usually do the same and recommend this stuff to people sometimes.

**Comment by cathugger** [comment]
merging then I guess

**Pull request merged**